package.json

```diff
@@ -14,17 +14,17 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@types/node-schedule": "^1.3.1",
     "bcrypt": "^5.0.1",
     "body-parser": "^1.19.0",
     "celebrate": "^14.0.0",
     "cookie-parser": "^1.4.5",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
     "jsonwebtoken": "^8.5.1",
+    "long-timeout": "^0.1.1",
     "mongodb": "^3.6.4",
     "mongoose": "^5.12.5",
-    "node-schedule": "^2.0.0",
+    "mongoose-function": "^0.1.0",
```

Comment:
This is really valuable to keep the project's dependencies as low as possible (you could imagine that you need to upgrade them each month and suffer from their vulnerabilities as well).
So could we consider replacing them with something?
`mongoose-function` is really old and `long-timeout` could be replaced using BigInt (I'm not sure).

Reply:
I don't think we can use BigInt instead of long-timeout as setTimeout accepts a number type. If we try to convert a BigInt to a number setTimeout will not work as it instantly executes if the value is greater than 2^31 - 1. Regarding mongoose-function, I think I basically replaced it with the equivalent. Mongoose-function stores the function as a string and then uses eval to execute it with some safety checking to make sure that the string is a function. As I don't need the safety checking due to the fact that only the server has access to this job manager, I just replaced it by storing the function as a string.

Reply:
Don't you already have a "posted at" timestamp in the DB? Why not only show posts that have that timestamp greater than now?

Reply:
Yup, I see. So these setTimeouts should be just chained. Okay, let's leave it with the library.